content/en/about/contributing/shortcodes/_index.md

```diff
@@ -144,102 +142,107 @@ in multiple programming languages, you can use a tabbed pane for code snippets:
 
 Will render as:
 
-{{< tabpane >}}
-{{< tab header="C++" lang="C++">}}
-#include <iostream>
-int main()
-{
-  std::cout << "Hello World!" << std::endl;
-}
-{{< /tab >}}
-{{< tab header="Java" lang="Java">}}
-class HelloWorld {
-  static public void main( String args[] ) {
-    System.out.println( "Hello World!" );
-  }
-}
-{{< /tab >}}
-{{< tab header="Kotlin" lang="Kotlin">}}
-fun main(args : Array<String>) {
-    println("Hello, world!")
-}
-{{< /tab >}}
-{{< tab header="Go" lang="Go">}}
-package main
-import "fmt"
-func main() {
-  fmt.Printf("Hello World!\n")
-}
-{{< /tab >}}
+{{< tabpane >}} {{< tab header="C++" lang="C++">}} #include <iostream> int
+main() { std::cout << "Hello World!" << std::endl; } {{< /tab >}}
+{{< tab header="Java" lang="Java">}} class HelloWorld { static public void main(
+String args[] ) { System.out.println( "Hello World!" ); } } {{< /tab >}}
+{{< tab header="Kotlin" lang="Kotlin">}} fun main(args : Array<String>) {
+println("Hello, world!") } {{< /tab >}} {{< tab header="Go" lang="Go">}} package
+main import "fmt" func main() { fmt.Printf("Hello World!\n") } {{< /tab >}}
 {{< /tabpane >}}
 
 ## Content with multiple variants
 
-When you need to include multiple variants of the same content, other than
-code snippets in multiple programing languages, you can use the follwing
-shortcode:
+When you need to include multiple variants of the same content, other than code
+snippets in multiple programing languages, you can use the follwing shortcode:
 
-```go-html-template
+````go-html-template
```

Comment:
Why 4 backticks?

Reply:
This is valuable information, maybe we should add it to the guide. E.g. I did not know this and had a hard time to make the examples render well.